index.d.ts

```diff
@@ -58,7 +58,7 @@ declare module 'react-table' {
     /**
      * This string/function is used to build the data model for your column.
      */
-    accessor: IdType<D> | AccessorFn<D>
+    accessor?: IdType<D> | AccessorFn<D>
```

Comment:
This works and allows header groups to be defined, but it doesn't help model the difference between headers and header groups particularly well.